recipes/qxlsx/all/conanfile.py

```diff
@@ -0,0 +1,77 @@
+import os
+from conans import CMake, ConanFile, tools
+
+required_conan_version = ">=1.43.0"
+
+
+class QXlsxConan(ConanFile):
+    name = "qxlsx"
+    description = "Excel file(*.xlsx) reader/writer library using Qt 5 or 6."
+    license = "MIT"
+    topics = ("qxlsx", "excel", "xlsx")
+    homepage = "https://github.com/QtExcel/QXlsx"
+    url = "https://github.com/conan-io/conan-center-index"
+
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False]
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True
+    }
+
+    generators = "cmake", "cmake_find_package_multi"
+    exports_sources = "CMakeLists.txt", "patches/*"
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+
+    def requirements(self):
+        self.requires("qt/5.15.2")
```

Comment:
if this library handles Qt6 as you said, why not depend on qt6?

Reply:
Why it should depend on qt6? KDE still uses qt5, a lot of apps also use qt5.

Reply:
To have a consistent dependency graph in conan-center.